rcl/include/rcl/client.h

```diff
@@ -370,6 +370,25 @@ RCL_WARN_UNUSED
 rmw_client_t *
 rcl_client_get_rmw_handle(const rcl_client_t * client);
 
+/// Check that the client is valid.
+/**
+ * The bool returned is `false` if client is invalid
+ * The bool returned is `true` otherwise.
+ * This function cannot fail.
+ *
+ * <hr>
+ * Attribute          | Adherence
+ * ------------------ | -------------
+ * Allocates Memory   | No
+ * Thread-Safe        | No
+ * Uses Atomics       | No
+ * Lock-Free          | Yes
+ *
+ * \param[in] client pointer to the rcl client
+ * \return `true` if `client` is valid, otherwise `false`
+ */
```

Comment:
Well it's always safe to leave the error message, but you do need to do it before calling something else if you don't want the overwriting an error message print. Elsewhere it is implied that a non `RCL_RET_OK` return code means an error message is set. Here it's not so clear that one would be set by a `false` return value. So I think it's appropriate.